net/natmap/Makefile

```diff
@@ -0,0 +1,42 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=natmap
+PKG_VERSION:=20221202
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/heiher/natmap.git
+PKG_SOURCE_DATE:=2022-11-11
+PKG_SOURCE_VERSION:=$(PKG_VERSION)
+PKG_MIRROR_HASH:=c41ee4a4525c59103e66ee96afac4a636728d714c3f6af4a25f38a71eff71bd9
```

Comment:
I wonder, why you are not download tarballs from tagged releases or using codeload at all?

Reply:
It would be nice if you can take a look at my previous review so you can get the answer.

Reply:
Hmm, I see, submodule, thanks for pointing it out! However, I think we can drop PKG_VERSION and rely on PKG_SOURCE_VERSION. Also, no need to have PKG_SOURCE_DATE, I will do suggestion soon.

Reply:
Good idea.

Reply:
What's the suggestion?

Reply:
imo this is the standard format and good enough, or you can drop `PKG_VERSION` and `PKG_SOURCE_DATE` so it saves 2 lines, several bytes hhh

Reply:
https://github.com/openwrt/packages/pull/19998#pullrequestreview-1202914938